index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+
+
+    <style type="text/css">
+      
+    </style>
+  </head>
+
+  <body>
+
+    <header>
+      <nav id="navbar">
+        <ul>
+          <li><a href="#welcome-section">About</a></li>
+          <li><a href="#projects">Work</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section id="welcome-section">
+      <h1 class="intro">Hey I am Mimic</h1>
+      <p class="title">a web developer</p>
+    </section>
+
+    <section id="projects">
+      <h2 id="project-head">These are some of my projects</h2>
+
+      <div class="project-grid" class="project-tile">
+        <div class="grid-child">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+
+
+        <div class="project-tile">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+
+
+        <div class="project-tile">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+
+
+        <div class="project-tile">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+
+        <div class="project-tile">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+
+        <div class="project-tile">
+          <div class="project-img"><img src="#"></div>
+          <div class="project-title"><a href="#">Project 1</a></div>
+        </div>
+      </div>
+
+      <div class="project-button">
+        <button>Show all ></button>
+      </div>
+    </section>
+
+    <section id="footer">
```

Comment:
Use footer element instead of section element.

Reply:
- Kindly try to make polite suggestions instead of orders and always try to give reasons for any change you recommend. It is an opportunity for the student to learn why they should use the footer elements. 
> ### Example response: 
> It would be better to use `<footer>` element instead of `<section>` element because `<footer>` is an HTML5 tag that adds semantics to your documents and is helpful for screen readers, keeping our code clean, and compatible with modern browser. 😊